packages/algob/src/lib/msig.ts

```diff
@@ -89,3 +89,31 @@ export function validateMsig (msig: MultiSig | undefined): void {
     throw new Error("Error fetching multisigned logic signature from file - invalid/undefined msig");
   }
 }
+
+/**
+ * Description: Signs an encoded signed multi-sig transaction object
```

Comment:
```suggestion
 * Signs an encoded signed multi-sig transaction object
```

Reply:
How about updating this to `Appends a signture`?

Reply:
Okay